### PR TITLE
Make the snippets game harder.

### PR DESCRIPTION
**Pull request by chucker**
It previously started out at 10%, and proceeded in increments of 10% each. Now it starts out at 3% and increments by 5%.

### CODE REVIEW

File: Prolangle/Services/SnippetRevealer.cs [hunk 1]
```diff
@@ -35,6 +33,8 @@ public string RevealMore()
 		concealedCode = concealedCode[..revealStart] + sourceCode.Substring(revealStart, revealEnd - revealStart) +
 		                concealedCode[revealEnd..];
 
+		RevealedProgress += 0.05;
```
Comment (chucker) [inline Prolangle/Services/SnippetRevealer.cs:36]: Note that this is a misnomer now. It's "progress to reveal next time".

Comment (ricardoboss) [reply Prolangle/Services/SnippetRevealer.cs:36]: That's fine. Maybe add a comment so we still know why it's at that position a year from now.

Comment (chucker) [reply Prolangle/Services/SnippetRevealer.cs:37]: Yeah.

**Comment by chucker** [comment]
Hmm. As the tests demonstrate, there's a bit of a problem here. In the tests, the snippet is just `27` chars. 3% of that is below one char, so the initial guess just shows… nothing.

I'm unsure if this means we should never have such a short snippet (probably), or if we need to add something that rounds up? (I think we effectively currently round _down_.)

**Comment by ricardoboss** [comment]
Yeah, use `max(5, length * progress)` as the number of revealed characters.

**Comment by ricardoboss** [comment]
Can you update the code to show at least 5 characters initially?

**Review by ricardoboss — CHANGES_REQUESTED** [review]
Can you change the revealer to show at least 5 characters initially? Regardless of the length of the snippet.

**Comment by chucker** [comment]
I've made some changes, but I'm not sure I've solved the underlying problem.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # _Waiting for deployment_
 
+* The snippets game has been made harder. Previously, you saw 10% of the code, and each guess gave you another 10%.for
```
Comment (ricardoboss) [inline CHANGELOG.md:3]: Typo. Also, please rebase

**Review by ricardoboss — APPROVED** [review]
LGTM

**Comment by chucker** [comment]
Oh, never mind. You already wrote "lgtm".

**Pull request merged**